tests/conftest.py

```diff
@@ -102,3 +105,43 @@ def form_element(db, form_entry):
     )
 
     return form_element
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def survey(db, location, study):
+    survey = Survey.objects.create(
+        time_stop=datetime.now(tz=None),
+        location=location,
+        study=study,
+        form_id=1,
+    )
+
+    return survey
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def survey_row(db, survey):
+    survey_row = SurveyRow.objects.create(
+        total=5,
+        survey=survey,
+    )
+
+    return survey_row
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def survey_component(db, row):
```

Comment:
Are `survey_row` and `survey_component` being used in the test at all, or are you setting them up for the future?

Reply:
Just setting them up for the future! I'll need them to test the next issue (#61)